src/main/java/org/stellar/sdk/Sep10Challenge.java

```diff
@@ -552,7 +552,14 @@ private static Set<String> verifyTransactionSignatures(Transaction transaction,
     }
 
     for (String signer : signers) {
-      KeyPair keyPair = KeyPair.fromAccountId(signer);
+      KeyPair keyPair;
+      try {
+        keyPair = KeyPair.fromAccountId(signer);
+      } catch (RuntimeException e) {
+        System.out.printf("Couldn't parse the signer \"%s\" as a valid Public Key.", signer);
```

Comment:
this console output is non-standard for the sdk which by default does not generate that type of output or logger outputs. So, to maintain consistency, would remove this and the stacktrace prints. the callers of this method could check the returned set, and determine which `signers` are missing, then it could deduce something was wrong and log something from the app layer, etc.